client/src/__locales/zh-cn.json

```
@@ -154,15 +154,15 @@
     "dns_settings": "DNS 设置",
     "dns_blocklists": "DNS 黑名单",
     "dns_allowlists": "DNS 白名单",
-    "dns_blocklists_desc": "AdGuard Home将阻止匹配DNS拦截清单的域名",
-    "dns_allowlists_desc": "来自 DNS 白名单的域名将被允许，即使它们位于任意黑名单中也是如此。",
+    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 拦截清单的域名",
```

```suggestion
    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 黑清单的域名",
```

The "blocklist" is translated as "黑名单" above. I think it's better to keep the consistency.

---

OK.